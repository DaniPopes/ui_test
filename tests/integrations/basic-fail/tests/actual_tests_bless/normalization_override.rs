//@normalize-stderr-test: "NORMALIZATION_OVERRIDE" -> "SUCCESS"

// Test that the above normalization will be run after the default normalization set in ui_tests_bless.rs

fn main() {
    asdf //~ ERROR: cannot find
}
