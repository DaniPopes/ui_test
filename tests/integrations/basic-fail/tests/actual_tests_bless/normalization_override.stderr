error[E0425]: cannot find value `asdf` in this scope
 --> tests/actual_tests_bless/SUCCESS.rs:6:5
  |
6 |     asdf
  |     ^^^^ not found in this scope

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0425`.
